features/canvas-2d-image-smoothingquality.yml

```
@@ -0,0 +1,6 @@
+name: Canvas 2D `imageSmoothingQuality` property
```

Maybe something like this? Though I'm a little concerned that this is not just a later addition to whatever feature `imageSmoothingEnabled` is a part of. It'd be nice to have some evidence of developers recognizing this as a distinct feature, or should have a reason to believe it so (e.g., blog posts and such promoting this to developers as a standalone thing).

```suggestion
name: imageSmoothingQuality for 2D canvases
```